<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создатель карт v0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelAngle">
            <property name="text">
             <string>Угол поворота:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="prefix">
             <string/>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelZoom">
            <property name="text">
             <string>Коефициент масштабирования, %:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
            <property name="minimum">
             <double>100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>200.000000000000000</double>
            </property>
            <property name="value">
             <double>101.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="MapView" name="graphicsView">
        <property name="dragMode">
         <enum>QGraphicsView::ScrollHandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionCreate"/>
    <addaction name="actionAddImg"/>
    <addaction name="actionSaveProj"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Поворот...</string>
     </property>
     <addaction name="actionLeft"/>
     <addaction name="actionRight"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Выровнять по...</string>
     </property>
     <addaction name="actionHorizontal"/>
     <addaction name="actionVertical"/>
    </widget>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="separator"/>
    <addaction name="menu_3"/>
    <addaction name="menu_6"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Формат</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionDegrees"/>
    <addaction name="actionMetres"/>
    <addaction name="separator"/>
    <addaction name="actionPoint1"/>
    <addaction name="actionPoint2"/>
    <addaction name="separator"/>
    <addaction name="actionZTop"/>
    <addaction name="actionZBottom"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Панель инструментов</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionClose">
   <property name="text">
    <string>Удалить выделенный элемент</string>
   </property>
   <property name="toolTip">
    <string>Удалить выделенный элемент</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть проект</string>
   </property>
   <property name="shortcut">
    <string>Shift+O</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить как изображение</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Увеличить элемент</string>
   </property>
   <property name="toolTip">
    <string>Увеличить элемент</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Уменьшить элемент</string>
   </property>
   <property name="toolTip">
    <string>Уменьшить элемент</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>Влево</string>
   </property>
   <property name="shortcut">
    <string>Shift+E</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Вправо</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionDegrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Градусы</string>
   </property>
  </action>
  <action name="actionMetres">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Метры</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionHorizontal">
   <property name="text">
    <string>Ширине</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="text">
    <string>Высоте</string>
   </property>
  </action>
  <action name="actionPoint1">
   <property name="text">
    <string>Задать точку 1</string>
   </property>
  </action>
  <action name="actionPoint2">
   <property name="text">
    <string>Задать точку 2</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>Закрыть все</string>
   </property>
  </action>
  <action name="actionSaveProj">
   <property name="text">
    <string>Сохранить проект</string>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionAddImg">
   <property name="text">
    <string>Добавить изображение</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="actionZTop">
   <property name="text">
    <string>Поднять на передний план</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionZBottom">
   <property name="text">
    <string>Опустить на задний план</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapView</class>
   <extends>QGraphicsView</extends>
   <header location="global">mapview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
